app/controllers/steps/base_step_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Steps
   class BaseStepController < ApplicationController
+    before_action :check_crime_application_presence
+
     # rubocop:disable Rails/LexicallyScopedActionFilter
```

Comment:
Do you think it might be worth removing this cop? Or do you think it will come in useful elsewhere?

Reply:
Oh, that cop is for the other `before_action` below that one, because rubocop is complaining that the actions :edit and :show are no declared in that controller (as this controller is suppose to be used in subclasses using inheritance).

I think ideally we might not have that many instances where this happen, so that's why I didn't disable this cop just yet unless it becomes too annoying. Another alternative is to declare the actions in this controller too, just empty or raise exception, mark as nocov and then implement in subclasses 🤔

Reply:
That might be clearer to have them declared there and it gives people a sign post of where they are perhaps 🤔

Reply:
Yep, I will change that in my next PR so it is more clear 👍